doc/source/install.rst

```diff
@@ -13,14 +13,39 @@ Windows.
 Requirements
 ------------
 
-nmrglue requires `NumPy <http://numpy.scipy.org>`_ and 
-`SciPy <http://www.scipy.org>`_ to be installed. The 
-`matplotlib <http://matplotlib.org/>`_ and `IPython <http://ipython.org/>`_
-packages are highly recommended.  A easy way of obtaining 
-and installing these packages is to use a Python distribution which provides 
-these packages, such as `EPD <http://www.enthought.com/products/epd.php>`_.  
-Detailed information on 
-`installing a Scipy stack <http://scipy.github.com/install.html>`_ is available.
+
+nmrglue dependes on the following open-source packages for scientific computing
+in the Python ecosystem.
+
++------------+------------+---------------------------------------+
+| Package    | Version    | Details                               |
++============+============+=======================================+
+| Python     | 2.7.9+     | Version 3 and above are recommended   |
++------------+------------+---------------------------------------+
+| Numpy      | 1.21+      | Required for all basic data types     |
```

Comment:
Numpy is only on 1.19.2. I'm not sure what the minimum version required for nmrglue is.  Maybe 1.16+ is a reasonable recommendation as that is the minimum version in NEP 29.